src/core/storage.ts

```diff
@@ -33,7 +33,21 @@ export class Storage {
           Vue.set(state, payload.key, payload.value)
         },
         RESET_RESOURCES (state) {
-
+          Vue.set(state.resources, 'new', {})
+          const resetCurrentIds = (obj) => {
```

Comment:
As with the checks for current IDs on subscribing to mercure topics - I'm sure this could be neater. We need to find current IDs twice in this update. Please do feel free to refactor if you see quick improvements